lib/id3c/labelmaker.py

```diff
@@ -184,6 +184,13 @@ class CollectionsCliaComplianceLayout(LabelLayout):
     reference = "scanpublichealth.org"
 
 
+class SamplesHaarviLayout(LabelLayout):
+    sku = "LCRY-2380"
+    barcode_type = "SAMPLE"
```

Comment:
Might want to specify `SERUM SAMPLE` (although not sure if there is enough room on the label...)

Reply:
Caitlin said it's not necessary to distinguish